codecov.yaml

```diff
@@ -0,0 +1,193 @@
+  - "autoware.universe/*/shape_estimation/::"
+  - "autoware.universe/*/shift_decider/::"
+  - "autoware.universe/*/signal_processing/::"
+  - "autoware.universe/*/simple_planning_simulator/::"
+  - "autoware.universe/*/stop_filter/::"
+  - "autoware.universe/*/surround_obstacle_checker/::"
+  - "autoware.universe/*/system_error_monitor/::"
+  - "autoware.universe/*/system_monitor/::"
+  - "autoware.universe/*/tensorrt_yolo/::"
+  - "autoware.universe/*/tier4_api_msgs/::"
+  - "autoware.universe/*/tier4_api_utils/::"
+  - "autoware.universe/*/tier4_auto_msgs_converter/::"
+  - "autoware.universe/*/tier4_autoware_launch/::"
+  - "autoware.universe/*/tier4_autoware_utils/::"
+  - "autoware.universe/*/tier4_calibration_rviz_plugin/::"
+  - "autoware.universe/*/tier4_control_launch/::"
+  - "autoware.universe/*/tier4_control_msgs/::"
+  - "autoware.universe/*/tier4_datetime_rviz_plugin/::"
+  - "autoware.universe/*/tier4_debug_msgs/::"
+  - "autoware.universe/*/tier4_debug_tools/::"
+  - "autoware.universe/*/tier4_external_api_msgs/::"
+  - "autoware.universe/*/tier4_hmi_msgs/::"
+  - "autoware.universe/*/tier4_integration_launch/::"
+  - "autoware.universe/*/tier4_localization_launch/::"
+  - "autoware.universe/*/tier4_localization_msgs/::"
+  - "autoware.universe/*/tier4_localization_rviz_plugin/::"
+  - "autoware.universe/*/tier4_map_launch/::"
+  - "autoware.universe/*/tier4_pcl_extensions/::"
+  - "autoware.universe/*/tier4_perception_launch/::"
+  - "autoware.universe/*/tier4_perception_msgs/::"
+  - "autoware.universe/*/tier4_perception_rviz_plugin/::"
+  - "autoware.universe/*/tier4_planning_launch/::"
+  - "autoware.universe/*/tier4_planning_msgs/::"
+  - "autoware.universe/*/tier4_planning_rviz_plugin/::"
+  - "autoware.universe/*/tier4_sensing_launch/::"
+  - "autoware.universe/*/tier4_simulation_msgs/::"
+  - "autoware.universe/*/tier4_simulator_launch/::"
+  - "autoware.universe/*/tier4_state_rviz_plugin/::"
+  - "autoware.universe/*/tier4_system_launch/::"
+  - "autoware.universe/*/tier4_system_msgs/::"
+  - "autoware.universe/*/tier4_v2x_msgs/::"
+  - "autoware.universe/*/tier4_vehicle_launch/::"
+  - "autoware.universe/*/tier4_vehicle_msgs/::"
+  - "autoware.universe/*/tier4_vehicle_rviz_plugin/::"
+  - "autoware.universe/*/time_utils/::"
+  - "autoware.universe/*/topic_state_monitor/::"
+  - "autoware.universe/*/traffic_light_classifier/::"
+  - "autoware.universe/*/traffic_light_map_based_detector/::"
+  - "autoware.universe/*/traffic_light_ssd_fine_detector/::"
+  - "autoware.universe/*/traffic_light_visualization/::"
+  - "autoware.universe/*/trajectory_follower/::"
+  - "autoware.universe/*/trajectory_follower_nodes/::"
+  - "autoware.universe/*/vehicle_cmd_gate/::"
+  - "autoware.universe/*/vehicle_constants_manager/::"
+  - "autoware.universe/*/vehicle_info_util/::"
+  - "autoware.universe/*/vehicle_velocity_converter/::"
+  - "autoware.universe/*/velodyne_monitor/::"
+  - "autoware.universe/*/web_controller/::"
```

Comment:
Could you explain the cause a little more concretely, please?
Since we don't want to maintain such a file, I'd like to find if there is another way.

Reply:
https://github.com/ros-tooling/action-ros-ci#integrate-action-ros-ci-with-codecov

I referred to the above link.
According to this, it seems that I have to modify the path for each package.
I tried the non-enumerated package approach, but it didn't work.

Reply:
If we do so, definitely we have to automate this.

Reply:
OK. I'll create a new workflow in autoware-github-actions. For now, I have set this PR to draft.